event_socket_server/base_server.py

```diff
@@ -27,6 +30,12 @@ def __init__(self, time, func, args, kwargs):
         self.cancel = False
         self.dispatched = False
 
+    def __eq__(a, b):
+        return not a < b and not b < a
```

Comment:
Dude... WHAT THE FUCK. `self.time == other.time` too hard. Also can you use `self` and `other` instead of `a` and `b`? Using anything but `self` for first argument is dumb.